docs/docs/api/path.md

```diff
@@ -7,8 +7,10 @@ sidebar_label: Path
 `Path` allow a parent view to give an arbitrary path to a child view at runtime.
```

Comment:
I guess this definition is a bit misleading as you could do something like this also:
```
foo = Path.bam.boo
baz = Observe[Arg.foo]
```
Also, you could potentially write all operations using the `Path` as well:
```
foo = Get.foo[Path.bam].baz[Path.boo.bap] // ~= Get.foo.bam.baz.boo.bap
```
So the definition of Path I guess would be more in the line of storing paths, making paths reusable or transmissible, and useful for general composition.